UNPRIVILEGED_INSTALL.md

```diff
@@ -0,0 +1,210 @@
+# Unprivileged Installation
+
+## 1 - Constraints
+
+The goal of this project is to install the Confluent platform under the following constraints:
+1. The user installing the software should not be privileged.
+2. The user installing the software should not have SSH access to the target hosts.
```

Comment:
I know this is your requirement, but is it a hard requirement of the feature?

Reply:
I don't think so. In this PR's modifications, we have 2 main things: (1) disabling privileged actions (mostly changing owners and rights) and (2) allowing the systemd user-mode. I haven't tested that yet, but I don't see a reason why it wouldn't work remotely.